tests/query_tests.rs

```diff
@@ -31,9 +31,6 @@ fn check_compilation(
         args.push("-u".to_string());
         args.push(format!("{}", udf_file.display()).to_string());
     }
-    let mut out_file = query_dir.join(query_name);
```

Comment:
Why remove these lines? They are important.

Reply:
The output file name is the same as the filter name argument.  That way you can define it at the command line without changing the name of all your queries.

Reply:
This leads to incorrect behavior. The reason for these two lines is to refresh the  *.cc files in the example_filters folder. Now they all are compiled to my_filter.cc, which is also wrong. For compilers standard practice is the `-o` flag to define the output object. The output file is also incorrectly named. It is going to be "filter..cc" instead of "filter.c". I think we may need tests for this.

Reply:
Okay.  I made the output filter's name depend on that argument, and will make some tests for it.